frontend/components/site/siteBuildLogs.jsx

```diff
@@ -12,27 +12,46 @@ import CommitSummary from './CommitSummary';
 export const REFRESH_INTERVAL = 15 * 1000;
 const BUILD_LOG_RETENTION_LIMIT = 180 * 24 * 60 * 60 * 1000; // 180 days in milliseconds
 
-function buildTooOld(build) {
-  return (new Date() - new Date(build.completedAt)) > BUILD_LOG_RETENTION_LIMIT;
+function shouldHaveBuildLogs(build) {
```

Comment:
this function (and the later logic relying on it) are pretty confusing patterns because they treat null and false differently. Is there a way to ensure we don't call this without a build?